plugin/trino-password-authenticators/src/main/java/io/trino/plugin/password/ldap/LdapAuthenticatorFactory.java

```diff
@@ -29,7 +29,7 @@
     @Override
     public String getName()
     {
-        return "ldap";
+        return "ldap-trino";
```

Comment:
Do we need this change ?

Reply:
It was requested earlier as it conflicts with the Presto LDAP Group Provider.

Reply:
I think it can be `ldap` so as to be consistent with other password-authenticator.